debian/changelog

```diff
@@ -1,24 +1,6 @@
-libnss-socket (0.2+gnoll3) lenny; urgency=low
+libnss-socket (0.1+gnoll3) lenny; urgency=low
```

Comment:
Actually, we can probably ditch this entire file if we're not planning to build debs...